lib/index.js

```diff
@@ -94,17 +135,23 @@ _applyPoints.forEach(function (applyPoint) {
 
             } else if (typeof routePolicy === 'function') {
 
+                if (!aggregateApplyPoint && routePolicy.runs && !routePolicy.applyPoint) {
+
+                    aggregateApplyPoint = determineAggregateApplyPoint(routePolicy.runs);
+                }
+
                 if (!hasValidApplyPoint(routePolicy)) {
```

Comment:
How does this work if the policy has no apply point, but indicates other policies to run with the .runs?